src/Compiler/Compiler/Latte2Compiler.php

```diff
@@ -50,7 +50,7 @@ public function compile(string $templateContent, ?string $actualClass, string $c
             $latteTokens = $parser->parse($templateContent);
             $className = $this->generateClassName();
             $phpContent = $compiler
-                ->setFunctions(array_keys($this->getDefaultFunctions()))
+                ->setFunctions(array_keys($this->getFunctions()))
```

Comment:
This was bug in Latte 2 implementation. We did not compiled with correctly set functions.